configs/fts-benchmarks.json

```diff
@@ -0,0 +1,109 @@
+            "concurrent_clients": 50,
+            "command": "FT.SEARCH rd0 \"__field:term__\""
+          },
+          {
+            "sub": "b",
+            "description": "Single term specific field",
+            "dataset": "datasets/search_terms.csv",
+            "concurrent_clients": 50,
+            "command": "FT.SEARCH rd0 \"@field1:__field:term__\""
+          },
+          {
+            "sub": "c",
+            "description": "Proximity all fields",
+            "dataset": "datasets/proximity_phrases.csv",
+            "concurrent_clients": 50,
+            "command": "FT.SEARCH rd0 \"__field:phrase__\""
+          },
+          {
+            "sub": "d",
+            "description": "Proximity specific inorder",
+            "dataset": "datasets/proximity_phrases.csv",
+            "concurrent_clients": 50,
+            "command": "FT.SEARCH rd0 \"@field1:__field:phrase__\""
+          },
+          {
+            "sub": "e",
+            "description": "Mixed pattern field-specific",
+            "dataset": "datasets/mixed_match.csv",
+            "concurrent_clients": 50,
+            "command": "FT.SEARCH rd0 \"@field1:__field:phrase__\""
+          },
+          {
+            "sub": "f",
+            "description": "Single term all fields + NOCONTENT",
+            "dataset": "datasets/search_terms.csv",
+            "concurrent_clients": 50,
+            "command": "FT.SEARCH rd0 \"__field:term__\" NOCONTENT"
+          },
+          {
+            "sub": "g",
+            "description": "Single term specific field + NOCONTENT",
+            "dataset": "datasets/search_terms.csv",
+            "concurrent_clients": 50,
+            "command": "FT.SEARCH rd0 \"@field1:__field:term__\" NOCONTENT"
+          },
+          {
+            "sub": "h",
+            "description": "Proximity all fields + NOCONTENT",
+            "dataset": "datasets/proximity_phrases.csv",
+            "concurrent_clients": 50,
+            "command": "FT.SEARCH rd0 \"__field:phrase__\" NOCONTENT"
+          },
+          {
+            "sub": "i",
+            "description": "Proximity specific inorder + NOCONTENT",
+            "dataset": "datasets/proximity_phrases.csv",
+            "concurrent_clients": 50,
+            "command": "FT.SEARCH rd0 \"@field1:__field:phrase__\" NOCONTENT"
+          },
+          {
+            "sub": "j",
+            "description": "Mixed pattern field-specific + NOCONTENT",
+            "dataset": "datasets/mixed_match.csv",
+            "concurrent_clients": 50,
+            "command": "FT.SEARCH rd0 \"@field1:__field:phrase__\" NOCONTENT"
+          }
+        ]
+      }
+    ],
+    "standalone_ingestion": [],
+    "duration": 60,
+    "clients": [10],
+    "pipelines": [1],
+    "warmup": 5,
+    "port": 6379,
+    "cluster_mode": false,
+    "tls_mode": false,
+    "server_cpu_range": "0-7",
+    "client_cpu_range": "8-15",
+    "io-threads": 8,
+    "profiling": {
+      "enabled": false,
```

Comment:
How does this profiling::enabled work? Does it get overriden by the script usage? Asking because the documentation mentions --profiling is an option